x-pack/plugins/licensing/public/license.test.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ILicense } from '../common/types';
+import { LICENSE_STATUS } from '../common/constants';
+import { LicenseFeature } from '../common/license_feature';
+import { setup } from './__fixtures__/setup';
+import { Plugin } from './plugin';
+
+describe('license', () => {
+  let plugin: Plugin;
+  let license: ILicense;
+
+  afterEach(async () => {
+    await plugin.stop();
+    sessionStorage.clear();
```

Comment:
I'm not sure what you mean by this, could you elaborate?

Reply:
optional: can we force clean `sessionStorage` between tests to prevent collisions / make sure testing environment is not affected by previous test set

Reply:
I'm confused, is that not what is happening here?

Reply:
it is. I'm asking that probably you know if we can automate such check